### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lefteris-L**

### CODE REVIEW

File: src/figuresClasses.ts [hunk 1]
```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+export type Shape = 'circle' | 'triangle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
```
Comment (ysrckr) [inline src/figuresClasses.ts:2]: Just a suggestion -> You can use `enum` type here.

**Review by ysrckr — APPROVED** [review]
LGTM 🤟🏻 Only one suggestion that would make it easier to expand the code.

File: src/figuresClasses.ts (hunk 1 above)
Comment (lefteris-L) [reply src/figuresClasses.ts:2]: Works like a charm. thanks

**Review by meforoma — APPROVED** [review]
charming 👍

**Pull request closed without merging**